costs/tags/tags_untagged.go

```diff
@@ -0,0 +1,170 @@
+//   Copyright 2017-2018 MSolution.IO
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package tags
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/trackit/jsonlog"
```

Comment:
Unordered tags

Reply:
They are disorderd, but they are the way `go fmt` formats them, for me we shouldn't change the default formatting

Reply:
We have been using this guideline since the very beginning, please follow it

Reply:
cf. comment below